bot/exts/evergreen/reverse.py

```diff
@@ -0,0 +1,41 @@
+from contextlib import suppress
+from typing import Optional
+
+from discord import AllowedMentions, Embed, Forbidden
+from discord.ext import commands
+
+from bot.utils import helpers
+
+
+class Reverse(commands.Cog):
+    """Cog for the reverse command."""
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(aliases=["rev", "ᓚᘏᗢ"])
+    async def reverse(self, ctx: commands.Context, *, text: Optional[str]) -> None:
+        """
+        Reverse the provided text
+        If no text is given then the users nickname is reversed.
+        """
+        if not text:
+            display_name = ctx.author.display_name
+
+            display_name = display_name[::-1]

```

Comment:
```suggestion
            display_name = ctx.author.display_name[::-1]
```

I mean... this works as well